Source/UTF8LineParser.swift

```diff
@@ -77,26 +62,9 @@ class UTF8LineParser {
         return lines
     }
 
-    func closeAndReset() -> [String] {
-        var lines: [String] = []
-
-        if seenCr {
```

Comment:
This was another simplification of flushing on carriage return immediately. Previously, `closeAndReset` would return up to two lines. One for a line that was already completed by a carriage return here, and the last unterminated line if it started on an invalid code-point.